site/docs/pages/getting-started.mdx

```diff
@@ -30,6 +30,48 @@ bun add @coinbase/onchainkit
 
 :::
 
+### API Key
+
+#### Obtain an API key from the [Coinbase Developer Platform APIs](https://portal.cdp.coinbase.com/products/onchainkit).
+
+<img
+  alt="OnchainKit copy API KEY"
+  src="https://onchainkit.xyz/assets/copy-api-key-guide.png"
+  width="auto"
+/>
+
+#### Create a .env file
+
+Create a new file in your project’s root directory and name it “.env”.
+
+<img
+  alt="OnchainKit copy API KEY"
+  src="https://onchainkit.xyz/assets/getting-started-create-env-file.png"
+  width="auto"
+/>
+
+#### Define your API KEY
```

Comment:
Define and Assign should be one step. I understand we are making stupid easy to understand how to do it. But I think we want to speak like midle school level, not elementary school level.

Reply:
Combined the stepss:

```
Define your API Key
Set your API key by assigning the value you copied from the Coinbase Developer Platform to the ONCHAINKIT_API_KEY variable.
```